bioio_sldy/reader.py

```diff
@@ -149,9 +150,14 @@ def _read_delayed(self) -> xr.DataArray:
         for timepoint in timepoints:
             data_for_timepoint: typing.List[da.Array] = []
             for channel in channels:
-                data = image.get_data(
+                value = dask.delayed(image.get_data)(
                     timepoint=timepoint, channel=channel, delayed=True
                 )
+                data = da.from_delayed(
+                    value,
+                    shape=(image.sizeZ, image.sizeY, image.sizeX),
+                    dtype=image.dtype,
+                )
```

Comment:
We were previously loading the whole file one timepoint+channel pair at a time??

Reply:
YES 😢 at least it was in memmap mode but still.... the test file is pretty big.  Each one of these calls had a noticeable hitch of less than a second but with the delayed code they return basically immediately.